### ISSUE TITLE
Please add PPA for Ubuntu 18.04

### ISSUE THREAD

**Issue by murbanowicz** [dotnet/core#1282]
Hi,
Is it possible to add PPA for Ubuntu 18.04 to let us install dotnet core sdk/runtime using apt-get ?

**Comment by richlander** [comment]
We're work on it. @leecow is working to make it happen. Totally agree!

**Comment by leecow** [comment]
Hey @murbanowicz - working with the team that manages packages.microsoft.com and hope to have an 18.04 feed up in the next few weeks.

**Comment by murbanowicz** [comment]
Any updated on that?

**Comment by Petermarcu** [comment]
@leecow , anything I can do to help on this one?

**Comment by leecow** [comment]
18.04 PPA is up. The next 2.1 Preview will be pushed there as well as an upcoming 2.0 update as soon as 18.04 support is added.

**Comment by Petermarcu** [comment]
Great. Should we upload Preview1 to it?

**Comment by leecow** [comment]
18.04 support was added to Master so I don't believe we have Preview 1 builds that would work. Certainly the tarball could be used along with a daily build of the [dependency installer](https://dotnetcli.blob.core.windows.net/dotnet/Runtime/master/dotnet-runtime-deps-latest-ubuntu.18.04-x64.deb) (or manually install them)

**Comment by Petermarcu** [comment]
Ok, @leecow do you need this issue open to track it happening or can we close it?

**Comment by damageboy** [comment]
OK... since 18.04 is two days away... is there an ETA on when the official non preview will be published for 18.04?

I'd be very happy to put my hands on `dotnet-sdk-2.1.105`... :)

**Comment by damageboy** [comment]
I just saw that dotnet-sdk-2.1.105 in the last few minutes...!:

```
$ lsb_release -d
Description:	Ubuntu Bionic Beaver (development branch)
$ dotnet --version
2.1.105
```
